### DISCUSSION TITLE
how to use scroll to element

### DISCUSSION THREAD

**Discussion by sageofagames** [Ginger-Automation/Ginger#2110] [Q&A] [answered]
I am using ginger 3.9 
trying to scroll to certain an element inside a svg element 

but i don't see how you can use the scroll to element action as it doesnt show where to scroll 

<img width="632" alt="2021-10-12 08_41_13-Ginger by amdocs" src="https://user-images.githubusercontent.com/50217974/136967264-d1ac063f-c41f-4168-a2df-15f9a1395bf5.png">

**Comment by sageofagames** [comment, marked as answer]
@JinendraGhodke

**Reply by JinendraGhodke** [reply]
In the locate by and locate value, specify the element on the page to which you want to scroll.
